core-plugins/src/main/java/io/cdap/plugin/batch/source/ExcelInputReader.java

```diff
@@ -90,7 +90,7 @@ public class ExcelInputReader extends BatchSource<LongWritable, Object, Structur
   private static final String RECORD = "record";
   private static final String EXIT_ON_ERROR = "Exit on error";
   private static final String WRITE_ERROR_DATASET = "Write to error dataset";
-  private static final String NULL = "NULL";
+  private static final String NULL = "";
```

Comment:
Please add a Unit Test to validate the null scenario.

Reply:
Discussed offline the UT for this does not run today, hence no change required here.